### PR TITLE
Remove entity name exists in workspace check from MDB while creating

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java [hunk 1]
```diff
@@ -276,6 +276,7 @@ private Project insertProject(Project project, String workspaceName, UserInfo us
   public Project updateProjectName(UserInfo userInfo, String projectId, String projectName)
       throws InvalidProtocolBufferException {
     try (Session session = ModelDBHibernateUtil.getSessionFactory().openSession()) {
+      // TODO: Remove this after UAC support update entity name using SetResource
```
Comment (anandjakhaniya) [inline backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java:278]: @conradoverta added TODO for how to update entity name at UAC??

**Review by conradoverta — COMMENTED** [review]
We're missing the change on project create, right?

**Comment by anandjakhaniya** [comment]
Yes. I was forget it. Pushed changes

File: backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java (hunk 1 above)
Comment (conradoverta) [reply backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java:278]: UAC already supports name update, but the question here is more how to ensure consistency between the two systems

**Review by conradoverta — APPROVED** [review]
LGTM assuming you have tested it

File: backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java:278]: I am trying to figure out how to update resource name at UAC. thread https://vertaai.slack.com/archives/C01JVHUHKR6/p1610115155062800

Comment (conradoverta) [reply backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java:278]: Not a blocker to merge this

**Pull request merged**